src/dmd/dsymbol.d

```diff
@@ -476,6 +476,9 @@ extern (C++) class Dsymbol : ASTNode
      * `toParent3()` returns a logically enclosing scope symbol this is a member of.
      * It skips over TemplateMixin's.
      *
+     * `toParent4()` similar to `toParent2()` but always follows the template declaration scope
+     * instead of the instantiation scope.
```

Comment:
I should make it clear that it ignores static templates

Reply:
I'll probably change the function to suit what the comment says and rewrite some code to accommodate for that t makes ore sense this way.